src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1544,21 +1544,11 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string,
     return;
   }
 
-  if (PTUIterationType::RetryIteration == iteration_type) {
-    uint32_t app_id_for_sending = GetAppIdForSending();
-
-    if (0 != app_id_for_sending) {
-      MessageHelper::SendPolicySnapshotNotification(
-          app_id_for_sending, pt_string, std::string(), application_manager_);
-    }
-
-  } else {
-    MessageHelper::SendPolicyUpdate(
-        policy_snapshot_full_path,
-        TimeoutExchangeSec(),
-        policy_manager_->RetrySequenceDelaysSeconds(),
-        application_manager_);
-  }
+  MessageHelper::SendPolicyUpdate(
```

Comment:
From PR description I can see that changes `Covered by Unit tests.` 

But I do not see any changes in Unit tests. 
If before changes all unit tests were passed, and after changes it still passed I would add additional unit test that should fail without appropriate changes, but pass with it.

Reply:
Hello @LuxoftAKutsan sorry, I have not done any changes in Unit tests. Please add additional unit test as you need.